README.md

```diff
@@ -1,12 +1,36 @@
 # Chainweb Relay App
 
-## Setup
+## What is it?
 
-Install dependencies
+This is a small server application to power Kadena bridges. The first iteration
+supports bridging transactions from Ethereum. 
+
+The application can:
+- Listen to Ethereum token contracts for transfers into a designated "locker" address.
+- For selected transfers, retreive header information from Ethereum in order to "propose" it to the relay smart contract on Kadena
+- Respond to proposals to validate headers by quering Ethereum and asserting they match. 
+
+## Who needs it?
+
+Bonders in the Kadena Chain Relay need to endorse enough headers in their bond lockup period in order to
+- earn a per-endorsement fee
+- qualify for risk fees in the form of APY
+
+Thus bonders either need to operate the software themselves, or have a service provider like Flux operate the software on their behalf.
+
+## Prerequisites for running the software
+
+A given instance of the software runs for a single bond, and must be configured with the private key that governs the bond.
+
+### Rotating the bond key
+
+Bonders who have purchased bonds on relay.chainweb.com using their KDA public key should NOT expose their public key to this software.
+Instead, bonders should **rotate their bond to a dedicated keypair just for bond administration** as this will protect their KDA account.
+
+Rotating can be achieved by 
+1. Generating a keypair
```

Comment:
How are bonders supposed to do this?

Reply:
A new keypair can be generated by
- clicking on `Generate Keypair` at https://transfer.chainweb.com/
- running a pact command `pact -g` for pact users.